my-conclusions.md

```diff
@@ -0,0 +1,14 @@
+My conclusions, Louise:
+
+Task 1:
+OOP side of the project is a lot less cluttered, making it easier to read.
+
+Task 2:
+Setting it up took a bit more time in OOP than in procedural, but readability is still better in OOP. Took about the same amount of lines of code.
+
+Task 3:
+Forgot to observe the changes, but was easier in OOP.
```

Comment:
@LouiseLM definitely. In oop you've introduced changes by adding a new code, not by modifying existing which usually is safer.